### PR TITLE
changes from new branch

### PR DESCRIPTION
**Pull request by prajakta00**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
     <h1>Hello</h1>
+    <p>Hi user</p>
```
Comment (Abhip32) [inline index.html:10]: This is new line added. Looks good

Comment (prajakta00) [reply index.html:10]: okay

File: index.html [hunk 2]
```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
     <h1>Hello</h1>
```
Comment (prajakta00) [inline index.html:9]: this looks good

Comment (Abhip32) [reply index.html:9]: OK resolving conversation

**Pull request merged**